.pre-commit-config.yaml

```diff
@@ -10,35 +10,8 @@ repos:
     -   id: check-yaml
     -   id: debug-statements
     -   id: end-of-file-fixer
-        exclude: "resources/.*|docs/make.bat"
     -   id: trailing-whitespace
     -   id: mixed-line-ending
         args: ['--fix=lf']
-        exclude: "docs/make.bat"
     -   id: check-added-large-files
         args: ['--maxkb=1000']
-# -   repo: https://github.com/pre-commit/mirrors-mypy
-#     rev: v0.942
-#     hooks:
-#     -   id: mypy
--   repo: https://github.com/psf/black
-    rev: 23.1.0
-    hooks:
-    -   id: black
-        args: [--target-version, py36]
--   repo: https://github.com/asottile/blacken-docs
-    rev: 1.13.0
-    hooks:
-    -   id: blacken-docs
-        additional_dependencies: [black==22.1.0]
```

Comment:
What was the reason to remove those?

Reply:
Given that this repository will be mostly dedicated to hosting Markdown files,
having 4 pre-commit hooks rgar lints Python code seemed a little over-cautious 😅

For example, slowing commits operations of all contributors in order to repeatdly check f 2 Python scripts are compatible wth Python 3.9 felt a bit unnecessary to me.

If you really want to keep them, I won't oppose that though 😊
There were some `exclude` rules that made me think this file was probably a copy & paste from another repository,
and you may not have taken the the time to trim down the number of hooks to the really necessery ones here.